### DISCUSSION TITLE
Incorrect values showing in kiali applications console

### DISCUSSION THREAD

**Discussion by kalyanteja19** [kiali/kiali#5515] [Q&A]
Hi Team,

We are using kiali 1.42 version and found that incorrect DR, gateway, virtualservice details are showing in the console.

http://localhost:20001/kiali/console/applications.

![image](https://user-images.githubusercontent.com/53315599/193244699-2330321b-e040-4daf-8065-b37f7d4dba3b.png)

**Comment by aljesusg** [comment]
Can you show me the details about the object? We can't check if this is a bug with this information.

**Comment by jmazzitelli** [comment]
I suspect this isn't a bug but a misunderstanding of how Kiali displays applications.

As alberto mentions, we need to see your resource details, particularly the names of the resources and their labels.

I'm converting this to a Q&A Discussion, as it most likely isn't a bug but just a question about what "application" means in the context of Istio and Kiali

**Reply by jmazzitelli** [reply]
I suggest you read this thread - particularly my responses with the analyses of label names and how it affects the use of the term "application" in Kiali:

https://github.com/kiali/kiali/discussions/5443#discussioncomment-3584492
